pip-freeze.txt

```diff
@@ -28,6 +28,7 @@ enum34==1.0.4
 flake8==2.5.4
 gunicorn==19.3.0
 hamlpy==0.82.2
+hypothesis==3.4.0
```

Comment:
We should probably ask before adding a requirement (mostly just flagging this so that people can object if needed).